test/suites/storage_buckets.sh

```diff
@@ -186,3 +186,113 @@ EOF
 
   delete_object_storage_pool "${poolName}"
 }
+
+test_storage_bucket_export() {
+  # shellcheck disable=2039,3043
+  local lxd_backend
+
+  lxd_backend=$(storage_backend "$LXD_DIR")
+
+  # Skip the test if we are not using a Ceph or Dir backend, because the test requires a storage pool
+  # larger than 1 GiB due to the Minio requirements: https://github.com/minio/minio/issues/6795
```

Comment:
why do we require 1GiB?

Reply:
Not sure why incus has that restriction, but tests pass for other backends so I've removed that check.